pandas/tests/frame/methods/test_to_dict.py

```diff
@@ -64,14 +64,24 @@ def test_to_dict_index_not_unique_with_index_orient(self):
         with pytest.raises(ValueError, match=msg):
             df.to_dict(orient="index")
 
-    def test_to_dict_invalid_orient(self):
+    @pytest.mark.parametrize("orient", ["xinvalid"])
```

Comment:
I think that parameteration is redundant, can you remove it and have this test as it was before

Reply:
yeah, agreed